app/views/users/show.html.erb

```diff
@@ -1,100 +1,81 @@
-<div class="container mt-4">
```

Comment:
I think I've taken all the styling from this page now. 
Need to make the message button work. 

Not going to merge the branch just yet.